adafruit_irremote.py

```diff
@@ -303,13 +313,13 @@ def _read_pulses_non_blocking(
 
     def read_pulses(
         self,
-        input_pulses,
+        input_pulses: list,
         *,
-        max_pulse=10000,
-        blocking=True,
-        pulse_window=0.10,
-        blocking_delay=0.10,
-    ):
+        max_pulse: int = 10000,
+        blocking: bool = True,
+        pulse_window: bool = 0.10,
+        blocking_delay: bool = 0.10,
```

Comment:
both `pulse_window` and `blocking_delay` look like they can be `float` instead of `bool`